### PR TITLE
new(content/en/docs): running Falco in a Kind cluster

### PR DESCRIPTION
**Pull request by leogr**
Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind transalation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

**What this PR does / why we need it**:

This PR adds instructions on how to properly run Falco in a Kind cluster.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/running.md [hunk 1]
```diff
@@ -44,6 +44,28 @@ By default, starting the container will attempt to load and/or build the Falco k
 docker run ... -e SYSDIG_SKIP_LOAD=1 ... falcosecurity/falco:{{< latest >}}
 ```
 
+## Running Falco in a Kind cluster
+
+The easiest way to run Falco on a [Kind](https://github.com/kubernetes-sigs/kind) cluster is as follows:
+
+1. Create a configuration file (e.g., `kind-config.yaml`) and add this content to the file:
```
Comment (radhikapc) [inline content/en/docs/running.md]: 1. Create a configuration file. For example: `kind-config.yaml`
2. Add the following to the file:

File: content/en/docs/running.md [hunk 2]
```diff
@@ -44,6 +44,28 @@ By default, starting the container will attempt to load and/or build the Falco k
...
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: /dev
+    containerPath: /dev
+```
+
+2. Then create the cluster specifying the configuration file:
```
Comment (radhikapc) [inline content/en/docs/running.md]: 3. Create the cluster by specifying the configuration file:

File: content/en/docs/running.md [hunk 3]
```diff
@@ -44,6 +44,28 @@ By default, starting the container will attempt to load and/or build the Falco k
...
+```
+kind create cluster --config=./kind-config.yaml
+```
+
+3. Now, you can [installed](../installation) Falco in your Kubernetes cluster running on Kind.
```
Comment (radhikapc) [inline content/en/docs/running.md]: 4. Ensure that Falco in your Kubernetes cluster is running on Kind.

(I changed the step to an action step)

**Comment by leogr** [comment]
> Minor suggestion. Otherwise great.

@radhikapc Thank you for your suggestion.
Done and also rebased to the current master.

**Pull request merged**